docs/development.md

```diff
@@ -233,3 +233,21 @@ This is GitHub user that does all the automation.
 | Script      | What it is doing                          | Access via                                    |
 |-------------|-------------------------------------------|-----------------------------------------------|
 | .travis.yml | Uploading binaries to GitHub release page | Personal access token `deploy-github-release` |
+
+
+## Licenses
+
+[wwhrd](https://github.com/frapposelli/wwhrd) is used in odo for checking license
+compatibilities of vendor-ed packages with that of Odo(Apache 2.0 licensed).
+
+Configuration for `wwhrd` is stored in
+[`.wwhrd.yml`](https://raw.githubusercontent.com/redhat-developer/odo/0e7941d3dd0dabef8d5b2c5ed9fcb16b0039ae90/.wwhrd.yml).
```

Comment:
this shouldn't point to the specific commit, to the master. Otherwise it will be confusing, as anyone who will follow this link will see old version

Reply:
Ack.. will change it
I put this link currently only to verify for now ;)